CITATION.cff

```diff
@@ -0,0 +1,26 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+  
+title: "US Arctic Observation Network Benefit Tool"
+doi: 10.5281/zenodo.8403760
```

Comment:
On the Zenodo page, there's a place on the right sidebar that says "Cite all versions?" and gives a special auto-updating DOI that you can use to cite all versions of this software. Let's use that one here (and in the DOI badge on the README).

If we use this DOI, both of those things will always reference this specific version (`0.5.0`), and we'd have to continuously update it to keep it in sync with each release. That sounds awful ;)